ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/snapsync/SnapWorldStateDownloadProcess.java

```diff
@@ -231,7 +230,7 @@ public SnapWorldStateDownloadProcess build() {
               .thenProcess(
                   "checkNewPivotBlock",
                   tasks -> {
-                    pivotBlockManager.check(blockHeader -> {});
+                    pivotBlockManager.check((___, __) -> {});
```

Comment:
core readabilty and remove duplication suggestion:
introduce something like `BiConsumer<BlockHeader, Boolean> doNothingOnPivotChange = (___, __) -> {};`  and use the variable as argument.

This also applies to the other occurencies below

Reply:
done